libs/framework/include/module.h

```diff
@@ -42,9 +43,9 @@ typedef struct module *module_pt;
 extern "C" {
 #endif
 
-module_pt module_create(manifest_pt headerMap, const char *moduleId, celix_bundle_t *bundle);
+module_pt module_create(manifest_pt headerMap, const char *moduleId, celix_framework_t* fw, celix_bundle_t *bundle);
```

Comment:
Why we need this extra fw if bundle already contains what we need?

I still need to understand the relationship between `module` and `bundle`.

Reply:
You are correct, removed the framework arg